dashboard/app/assets/stylesheets/application.scss

```diff
@@ -538,6 +539,22 @@ $default-modal-width: 640px;
       float: left;
       position: relative;
     }
+
+    .hint {
+      padding: 15px 15px 15px 40px;
+      background-color: $hint-bgcolor;
+      border-radius: 10px;
+    }
+    .hintInner {
+      padding: 15px;
+      background-color: $white;
+      border-radius: 10px;
+    }
+
+    img {
+      border-radius: 10px;
```

Comment:
what effect does this have on existing images like the (soon-to-be-deprecated) angle helper gif and the Star Wars inline images?